engines/stark/visual/text.h

```diff
@@ -62,6 +63,9 @@ class VisualText : public Visual {
 	void createTexture();
 	void freeTexture();
 
+	// Check whether the text is blank
```

Comment:
Please use Doxygen style comments when documenting methods/classes (so they appear in http://doxygen.residualvm.org/).